includes/table/class-following.php

```diff
@@ -0,0 +1,236 @@
+
+		if ( isset( $_GET['order'] ) ) {
+			$args['order'] = sanitize_text_field( wp_unslash( $_GET['order'] ) );
+		}
+
+		if ( isset( $_GET['s'] ) ) {
+			$args['s'] = sanitize_text_field( wp_unslash( $_GET['s'] ) );
+		}
+		$following_with_count = FollowingCollection::get_following_with_count( $this->user_id, $per_page, $page_num, $args );
+		$following            = $following_with_count['following'];
+		$counter              = $following_with_count['total'];
+
+		$this->items = array();
+		$this->set_pagination_args(
+			array(
+				'total_items' => $counter,
+				'total_pages' => ceil( $counter / $per_page ),
+				'per_page'    => $per_page,
+			)
+		);
+
+		foreach ( $following as $follow_id ) {
+			$actor = Actors::get_actor( $follow_id );
+			$item  = array(
+				'icon'       => \esc_attr( $actor->get_icon()['url'] ?? '' ),
+				'post_title' => \esc_attr( $actor->get_name() ),
+				'username'   => \esc_attr( $actor->get_preferred_username() ),
+				'url'        => \esc_attr( object_to_uri( $actor->get_url() ) ),
+				'identifier' => \esc_attr( $actor->get_id() ),
+				'published'  => \esc_attr( $actor->get_published() ),
+				'modified'   => \esc_attr( $actor->get_updated() ),
+			);
+
+			$this->items[] = $item;
+		}
+	}
+
+	/**
+	 * Returns bulk actions.
+	 *
+	 * @return array
+	 */
+	public function get_bulk_actions() {
+		return array(
+			'delete' => \__( 'Unfollow', 'activitypub' ),
+		);
+	}
+
+	/**
+	 * Column default.
+	 *
+	 * @param array  $item        Item.
+	 * @param string $column_name Column name.
+	 * @return string
+	 */
+	public function column_default( $item, $column_name ) {
+		if ( ! array_key_exists( $column_name, $item ) ) {
+			return \__( 'None', 'activitypub' );
+		}
+		return $item[ $column_name ];
+	}
+
+	/**
+	 * Column avatar.
+	 *
+	 * @param array $item Item.
+	 * @return string
+	 */
+	public function column_avatar( $item ) {
+		return sprintf(
+			'<img src="%s" width="25px;" alt="" />',
+			$item['icon']
+		);
+	}
+
+	/**
+	 * Column url.
+	 *
+	 * @param array $item Item.
+	 * @return string
+	 */
+	public function column_url( $item ) {
+		return sprintf(
+			'<a href="%s" target="_blank">%s</a>',
+			esc_url( $item['url'] ),
+			$item['url']
+		);
+	}
+
+	/**
+	 * Column cb.
+	 *
+	 * @param array $item Item.
+	 * @return string
+	 */
+	public function column_cb( $item ) {
+		return \sprintf( '<input type="checkbox" name="following[]" value="%s" />', esc_attr( $item['identifier'] ) );
+	}
+
+	/**
+	 * Process action.
+	 */
+	public function process_action() {
+		if ( ! isset( $_REQUEST['following'] ) || ! isset( $_REQUEST['_wpnonce'] ) ) {
+			return;
+		}
+		$nonce = \sanitize_text_field( \wp_unslash( $_REQUEST['_wpnonce'] ) );
+		if ( ! \wp_verify_nonce( $nonce, 'bulk-' . $this->_args['plural'] ) ) {
+			return;
+		}
+
+		if ( ! \current_user_can( 'edit_user', $this->user_id ) ) {
+			return;
+		}
+
+		$following = $_REQUEST['following']; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
```suggestion
		$following = \sanitize_url( \wp_unslash( $_REQUEST['following'] ) );
```
Would this be a good way to sanitize this value?